yeat/workflow/Pacbio

```diff
@@ -45,12 +45,12 @@ rule canu:
 
 rule hifiasm:
     output:
-        contigs="analysis/{sample}/{readtype,pacbio-raw|pacbio-corr|pacbio-hifi}/{label}/hifiasm/contigs.fasta"
+        contigs="analysis/{sample}/pacbio-hifi/{label}/hifiasm/contigs.fasta"
     input:
-        reads="seq/input/{sample}/{readtype}/combined-reads.fq.gz"
+        reads="seq/input/{sample}/pacbio-hifi/combined-reads.fq.gz"
     threads: 128
     params:
-        prefix="analysis/{sample}/{readtype}/{label}/hifiasm/{sample}",
+        prefix="analysis/{sample}/pacbio-hifi/{label}/hifiasm/{sample}",
         extra_args=lambda wildcards: config["assemblies"][wildcards.label].extra_args
```

Comment:
`Hifiasm`, `Hifiasm-meta`, and `MetaMDBG` are tested and primarily designed for PacBio HiFi reads. In the "Supported Read Types and Algorithms" table, users can see that these assembly algorithms are compatible exclusively with PacBio HiFi reads.

Made the following changes to reflect that in the Snakemake rules.

https://github.com/bioforensics/yeat/wiki